src/App.tsx

```diff
@@ -1,26 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
-
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+import { Pagination } from './components/Pagination';
 
 export const App: React.FC = () => {
+  const [state, setState] = useState({
+    perPage: 5,
+    currentPage: 1,
+  });
+
+  const arrOfItemsForCurrPage = (
```

Comment:
Get used to name functions according to these rules:
![image](https://user-images.githubusercontent.com/96409563/229037981-c217ed9c-300b-4d87-8e43-644a5a8b05e9.png)